extensions/azure/assetindex-cosmos/src/main/java/org/eclipse/dataspaceconnector/assetindex/azure/AssetIndexCosmosConfig.java

```diff
@@ -0,0 +1,53 @@
+package org.eclipse.dataspaceconnector.assetindex.azure;
+
+import org.eclipse.dataspaceconnector.cosmos.azure.AbstractCosmosConfig;
+import org.eclipse.dataspaceconnector.spi.EdcSetting;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+
+public class AssetIndexCosmosConfig extends AbstractCosmosConfig {
+
+    @EdcSetting
+    private static final String COSMOS_ACCOUNTNAME_SETTING = "edc.assetindex.cosmos.account-name";
+    @EdcSetting
+    private static final String COSMOS_DBNAME_SETTING = "edc.assetindex.cosmos.database-name";
+    @EdcSetting
+    private static final String COSMOS_PARTITION_KEY_SETTING = "edc.assetindex.cosmos.partition-key";
+    @EdcSetting
+    private static final String COSMOS_PREFERRED_REGION_SETTING = "edc.assetindex.cosmos.preferred-region";
+    @EdcSetting
+    private static final String COSMOS_CONTAINER_NAME_SETTING = "edc.assetindex.cosmos.container-name";
```

Comment:
Maybe you can add a little README.md that lists the existing configuration parameters. So that somebody, that wants to use this extension, just needs to look through the README.md.

Reply:
Done, you can find the file here: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/192/files#diff-d4164de6f93c8b9c67bd2c9239ba2437da6c7cd725690ca89f470542bc456e75